### ISSUE TITLE
As observer you should be able to toggle a clientside personal pointlight

### ISSUE THREAD

**Issue by UbaserB** [space-wizards/space-station-14#33486]
## Description

On goon they have this feature and its so useful, keeps the atmosphere/vibe while being able to see stuff

**Screenshots**

![image](https://github.com/user-attachments/assets/239df7c5-5891-4464-b9b6-4e7bf9f829cb)
